v3/src/components/graph/adornments/plotted-value/plotted-value-model.ts

```diff
@@ -0,0 +1,34 @@
+import { Instance, types } from "mobx-state-tree"
+import { mean } from "mathjs"
+import { AdornmentModel, IAdornmentModel, IUpdateCategoriesOptions } from "../adornment-models"
+import { kPlottedValueType } from "./plotted-value-types"
+
+export const PlottedValueModel = AdornmentModel
+  .named("PlottedValueModel")
+  .props({
+    type: "Plotted Value",
+    value: types.maybe(types.union(types.number, types.string))
```

Comment:
For the plotted value, the formula should be serialized, not the value. It may be that all you need is an instance of `FormulaModel`. Probably okay to leave this for @pjanik to work out as well.

Reply:
I spent a little time on this but it wasn't clear to me how to do it. So unless @pjanik feels strongly otherwise, I'm going to leave it for him.